concent_api/core/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db.models import Model, CharField, IntegerField, DateTimeField, BooleanField, BinaryField, ForeignKey
```

Comment:
Co sądzicie o robieniu importów taki sposób (oraz też alfabetycznie):
from django.db.models import (
    CharField, 
    IntegerField,
    Model, 
    ...
)

Z tego co mówiliście zależy Wam na historii w gicie, a tutaj widać dokładnie kto dodał jaką linijkę z importem (przy jednej linijce to się nadpisuje).

Reply:
I'd rather do 

``` python
from a                import baz
from django.db.models import CharField
from django.db.models import IntegerField
from django.db.models import Model
from foo.bar          import baz
from x                import y

from .models          import Message
...
```

which is IMO makes it easier to visually separate the import block at the beginning of the file from the rest.

I mostly try to sort the imports alphabetically but only when it comes to modules, not individual things imported from them. But pylint does not complain about it so it's not strictly enforced.

But I do like the idea of one import per line and I even considered switching to this style it in the past. Multiple imports on a single line are often a source of trivial conflicts when rebasing. I'm for switching to that style.

Reply:
:+1: